pkg/ipam/poolallocator/allocator.go

```diff
@@ -152,19 +153,55 @@ func (a *IPPoolAllocator) appendPoolUsage(ipPool *v1alpha2.IPPool, ip net.IP, st
 
 }
 
-// Update pool status to delete released IP
+func (a *IPPoolAllocator) updatePoolUsage(ipPool *v1alpha2.IPPool, ip net.IP, state v1alpha2.IPAddressPhase, owner v1alpha2.IPAddressOwner) error {
```

Comment:
Please add comment, for example, the method expects ipPool to contain the IP.

Reply:
Added.